data/json/monster_special_attacks/spells.json

```diff
@@ -586,6 +586,120 @@
     "min_field_intensity": 1,
     "max_field_intensity": 2
   },
+  {
+    "type": "SPELL",
+    "id": "spell_gene_sting",
+    "name": { "str": "Gene Sting" },
+    "description": "Yugg attack which causes random mutation.",
+    "effect": "attack",
+    "shape": "blast",
+    "valid_targets": [ "hostile" ],
+    "min_range": 24,
+    "max_range": 24,
+    "sound_description": "ssszziiipp",
+    "min_damage": 1,
+    "max_damage": 1,
+    "extra_effects": [ { "id": "spell_gene_sting_selector" } ]
+  },
+  {
+    "id": "spell_gene_sting_selector",
+    "type": "SPELL",
+    "name": "Gene Sting Selector",
+    "description": "The spell fires the EOC for the yugg gene sting.  It's a bug if you have it.",
+    "message": "",
+    "valid_targets": [ "hostile" ],
+    "flags": [ "NO_FAIL", "SILENT", "NO_HANDS", "NO_LEGS" ],
+    "effect": "effect_on_condition",
+    "effect_str": "EOC_sting_selector",
+    "shape": "blast",
+    "max_level": 1
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_random_mutate",
+    "condition": "u_is_character",
+    "effect": [ { "set_string_var": "<random_category>", "target_var": { "context_val": "dart_mutation_category" }, "parse_tags": true },
+{ "u_mutate_category": { "context_val": "dart_mutation_category" } } ],
+"snippet_category": [
```

Comment:
@GuardianDll is this one or the one below the right way to do what you were talking about?